.github/workflows/flatpak.yml

```diff
@@ -68,7 +68,7 @@ jobs:
           echo "::set-output name=commitHash::$(git rev-parse --short=9 HEAD)"
 
       - name: Build Flatpak Manifest
-        uses: bilelmoussaoui/flatpak-github-actions/flatpak-builder@master
+        uses: flatpak/flatpak-github-actions/flatpak-builder@master
```

Comment:
This one should maybe be reverted to v4.

Reply:
Same here